### DISCUSSION TITLE
Image of docker Building Environment Available for arm64

### DISCUSSION THREAD

**Discussion by txmu** [termux/termux-packages#8544] [Ideas]
I want to build a software package for termux on an arm64 high-performance server...

**Comment by ghost** [comment]
This isn't possible. Our build environment depends on Android NDK which is available only for x86_64.

**Reply by txmu** [reply]
So, may I use qemu-user to do this? How do you do it?

**Reply by ghost** [reply]
You can, though QEMU will discard all performance of your "high performance arm64 server".

> How do you do it?

No, I don't use QEMU and don't need either. Packages are cross-compiled.

**Reply by txmu** [reply]
> https://github.com/suhan-paradkar/TermuSDK

> https://github.com/Lzhiyong/termux-ndk



Can these help me?

**Reply by ghost** [reply]
They are for Termux and won't work on arm64 Linux distro like apps from Linux distro doesn't work on Termux. And we support only official build of NDK revision 23, everything else isn't guaranteed to work.
